code/controllers/subsystem/air.dm

```diff
@@ -380,6 +388,18 @@ SUBSYSTEM_DEF(air)
 		AM.build_network()
 		CHECK_TICK
 
+GLOBAL_LIST_EMPTY(colored_turfs)
+GLOBAL_LIST_EMPTY(colored_images)
+/datum/controller/subsystem/air/proc/setup_turf_visuals()
+	for(var/sharp_color in GLOB.contrast_colors)
+		var/obj/effect/overlay/atmos_excited/suger_high = new()
+		GLOB.colored_turfs += suger_high
+		var/image/shiny = new('icons/effects/effects.dmi', suger_high, "atmos_top", ATMOS_GROUP_LAYER)
+		shiny.plane = ATMOS_GROUP_PLANE
+		shiny.color = sharp_color
+		GLOB.colored_images += shiny
+		CHECK_TICK
```

Comment:
Why would you need to check tick this, it's 5 images.

Reply:
I'm paranoid